src/MongoDB.Driver/Linq/Translators/FindProjectionTranslator.cs

```diff
@@ -89,7 +89,9 @@ private FindProjectionTranslator(DocumentExpression documentExpression, Paramete
 
         protected internal override Expression VisitField(FieldExpression node)
         {
-            if (!_fields.Any(x => x.FieldName == node.FieldName
+            var fieldExpression = GetFormattedFieldExpression(node);
```

Comment:
Minor renaming suggestion for both the variable and the method names:

    var dottedFieldExpression = GetDottedFieldExpression(node);

Reply:
Done